src/lib/tabwidget/tabstackedwidget.cpp

```diff
@@ -262,6 +262,8 @@ int TabStackedWidget::pinUnPinTab(int index, const QString &title)
 
     bool makePinned = index >= m_tabBar->pinnedTabsCount();
     QWidget* button = m_tabBar->tabButton(index, m_tabBar->iconButtonPosition());
+    // To show tooltip of tab which is pinned in the current session
+    QString m_tabToolTip = tabToolTip(index);
```

Comment:
`m_` prefix is only for member variables, so in this case it should just be `tabToolTip`.